runtime/doc/lua.txt

```diff
@@ -668,8 +668,8 @@ vim.regex:match_line({bufnr}, {line_idx}, {start}, {end_})
     Parameters: ~
       • {bufnr}     (integer)
       • {line_idx}  (integer)
-      • {start}     (integer|nil)
-      • {end_}      (integer|nil)
+      • {start}     integer?
+      • {end_}      integer?
```

Comment:
We should make sure this is still surrounded with parens.

Reply:
I've updated the PR with these changes.